website/source/docs/guides/forwarding.html.md

```diff
@@ -155,6 +156,32 @@ You may have to add the following line to the bottom of your
 include: "/etc/unbound/unbound.conf.d/*.conf"
 ```
 
+### systemd-resolved Setup
+
+systemd-resolved is typically configured with `/etc/systemd/resolved.conf`. 
+To configure systemd-resolved to send queries for the consul domain to
+Consul, configure resolved.conf to contain the following:
```

Comment:
nitpick: I'd put `systemd-resolved` in code fences as it reads a bit funny if it's not clearly the name of a binary. Might not be consistent with other docs here I guess...